src/crystal/system/win32/file_descriptor.cr

```diff
@@ -75,7 +69,8 @@ module Crystal::System::FileDescriptor
     bytes_written
   end
 
-  private def system_blocking?
+  # :nodoc:
+  def system_blocking?
     @system_blocking
   end
```

Comment:
Could be shortened to `getter? system_blocking`

Reply:
That's an unrelated change, so I won't address it in this PR.